src/main/java/com/openlattice/chronicle/services/surveys/SurveysService.java

```diff
@@ -966,7 +966,7 @@ public void submitTimeUseDiarySurvey(
             Map<UUID, Map<UUID, Map<UUID, Set<Object>>>> entitiesByEntitySet = getEntitiesByESID( entityKeyIdMap,
                     entitiesByEntityKey );
             entitiesByEntitySet.forEach( ( entitySetId, groupedEntities ) -> {
-                dataApi.updateEntitiesInEntitySet( entitySetId, groupedEntities, UpdateType.PartialReplace );
+                dataApi.updateEntitiesInEntitySet( entitySetId, groupedEntities, UpdateType.Merge );
```

Comment:
Why are you making this change? What does it fix?

Reply:
Actually this change was unnecessary.